src/Magento/FunctionalTestingFramework/Console/DoctorCommand.php

```diff
@@ -0,0 +1,206 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types = 1);
+
+namespace Magento\FunctionalTestingFramework\Console;
+
+use Codeception\Configuration;
+use Symfony\Component\EventDispatcher\EventDispatcher;
+use Codeception\SuiteManager;
+use Magento\FunctionalTestingFramework\Config\MftfApplicationConfig;
+use Magento\FunctionalTestingFramework\Exceptions\TestFrameworkException;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Magento\FunctionalTestingFramework\Util\ModuleResolver;
+use Magento\FunctionalTestingFramework\Module\MagentoWebDriver;
+use Magento\FunctionalTestingFramework\Module\MagentoWebDriverDoctor;
+
+class DoctorCommand extends Command
+{
+    const CODECEPTION_AUTOLOAD_FILE = PROJECT_ROOT . '/vendor/codeception/codeception/autoload.php';
+    const MFTF_CODECEPTION_CONFIG_FILE = ENV_FILE_PATH . 'codeception.yml';
+    const SUITE = 'functional';
+    const COLOR_LIGHT_GREEN = "\e[1;32m";
+    const COLOR_LIGHT_RED = "\e[1;31m";
+    const COLOR_LIGHT_DEFAULT = "\e[1;39m";
+    const COLOR_RESTORE = "\e[0m";
```

Comment:
Please remove those constants and make use of `\Symfony\Component\Console\Style\SymfonyStyle` class.
here is more info how it work: https://symfony.com/doc/current/console/style.html

Reply:
Changed. However, the code does style the output but there is no color change in my environment.